eng/pipelines/common/templates/runtimes/run-test-job.yml

```diff
@@ -211,7 +211,6 @@ jobs:
       - script: $(_msbuildCommand)
                 $(Build.SourcesDirectory)/src/mono/mono.proj
```

Comment:
I think you're missing to pass down the configuration and maybe other parameters like os and arch. Also, why did you remove the `-ci` flag?

Reply:
Msbuild told me it did not recognize the -ci argument when I tried to use it; does it take some different equivalent argument?

Reply:
Oh, so it looks like $(_msbuildCommand) actually refers to a shell script:

./eng/common/msbuild.sh --ci 

So the --ci option is automatically there if you use $(_msbuildCommand) (and everything else correctly detects it is running in CI correctly).

Reply:
Awesome. Thanks for checking 😄